Rakefile

```
@@ -9,4 +11,47 @@ begin
+    puts "Generating API client..."
+    # Using a hardcoded version for now - we can update this later to read from the gemspec
+    version = "1.0.0"
+    properties = [
+      "gemName=kinde_api",
+      "gemVersion=#{version}",
+      "gemLicense=MIT",
+      "gemHomepage=https://github.com/kinde-oss/kinde-ruby-sdk",
+      "gemDescription='Ruby SDK for the Kinde Management API'",
+      "gemAuthor=Kinde",
+      "gemAuthorEmail=support@kinde.com",
+      "useAutoload=true"
```

RuboCop (1.75.5)

[convention] Put a comma after the last item of a multiline array.

(Style/TrailingCommaInArrayLiteral)